.github/workflows/release-notes.py

```diff
@@ -19,7 +19,7 @@
 Run a single node as follows:
 ```
 docker pull rqlite/rqlite
-docker run -p4001:4001 rqlite/rqlite
+docker run -d -p 4001:4001 -v data:/rqlite rqlite/rqlite
```

Comment:
Can you explain why we should enable this flags?

Reply:
Thus, when I ran this container, I saw logs that apparently information such as snapshots are stored in the /rqlite path, so I preferred to store this data in a volume.
And I used the -d flag to run this container in the background.
![240517_19h15m25s_screenshot](https://github.com/rqlite/rqlite/assets/116190942/2529de4a-ad3d-4528-8d9f-8d1dfb60cba6)

Reply:
OK, so at least it's good to add this to the documentation that important data is in the /rqlite path.